index.js

```diff
@@ -115,6 +115,7 @@ function ParseServer(args) {
   router.merge(require('./installations'));
   router.merge(require('./functions'));
   router.merge(require('./schemas'));
+  router.merge(require('./global_config'));
```

Comment:
The actual route is handled with `GET /config` and `POST /config` (it's defined in `global_config.js`) but I just haven't been able to call the actual file `config.js` because that one already existed and I didn't want to rename all of that.

With regards to a flag .. shouldn't I not just implement the `PUT` and `DELETE` routes if we want those, then? However I'm not sure how we would like to handle those because as I see it a given Parse instance only contain _one_ config and hence a `DELETE` doesn't really make sense to me. Same goes for `PUT` since this is handled by `POST`.